### DISCUSSION TITLE
[Feature Request]: Match post_max_size to upload_max_filesize

### DISCUSSION THREAD

**Discussion by Ratko-Solaja** [beyondcode/herd-community#362] [Ideas]
### Feature Description

I have used Max File Upload Size feature to set my desired upload size, however, WordPress in Site Health, tells me that the `post_max_size` is less than `upload_max_filesize`.

Maybe, setting Max File Upload Size (`upload_max_filesize`) can also overide the `post_max_size` to match the value.

**Comment by jannisborgers** [comment]
I’d love that. My CMS complains about the maximum upload size being 8M. The culprit is `post_max_size` though, not `upload_max_filesize`, which I set to 256 MB already.

**Comment by tyroga** [comment]
Yes, agree, this needs to be done as it seems changing it in the php.ini doesn't seem to fix it for me. 😓

**Comment by jannisborgers** [comment]
Changing `post_max_size` in all .ini files for Herd's PHP (clicking Show .ini file in menu bar), then closing and opening Herd did the trick for me.

**Comment by snapey** [comment]
AFAIK the post_max_size should be at least 30% more than the upload_max_filesize because of the base64 encoding of the binary image file.

I don't see the point of a box that allows changing of one and not the other as the user still needs to edit the php.ini file.

Either add post_max_size to the settings page or remove the upload size.

**Comment by kobe-ra** [comment]
I changed these variables in all `php.ini` files, but it still doesn't matter as the nginx in blocking the requests.
I'm getting 413 Request Entity Too Large
nginx/1.25.4

Could we also access `/etc/nginx/nginx.conf` in Herd? I don't know where it is as `/etc/nginx` doesn't exist with Herd installed.

**Comment by amdkholil** [comment]
i got this error now with php  8.3, add `post_max_size` in php.ini not work, only working for `upload_max_filesize`

**Comment by ynamite** [comment]
Adding `post_max_size=200M` to php.ini did the trick. Being able to change this in the GUI would be awesome though.
